### ISSUE TITLE
Posts that get hidden, don't stay hidden

### ISSUE THREAD

**Issue by cinnamennen** [christianselig/apollo-bugs#217]
**Describe the bug**
For quite some time now, even though I've done the "mark posts above as hidden" dance, they keep coming back.

**To Reproduce**
1. Scroll down a bunch
2. Click on '...', then hide posts above 
3. Pull down to refresh
4. See posts again that were "hidden"

**iOS version**: Version of iOS this bug occurred on (can be found in Settings.app > General)
**Apollo version**: Version of Apollo this bug occurred on (can be found in Apollo's Settings tab > About)
**iOS device**: For example iPhone XS, iPad mini, etc.
**Jailbroken**: Whether or not your device is jailbroken
**Reproducible:** Can you reproduce it every time? Or is it random?

**Expected behavior**

**Screenshots**
If possible, add screenshots to help show the problem.

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**

**Comment by yoBoss283** [comment]
Yes, also, sometimes after step 3 all post are hidden. The next time I open Apollo some of the.pm come back and some not. I noticed that when I click on “hide all above” the iPad start this circle sign next to the WiFi icon (meaning it is sending data), and hat takes time. Not sure if the API for hiding post takes so long. While this happens, one can still go to a post and start doing other actions, I am not sure if that cancels the previous “hide all above” command. This might explains also why sometimes some post are hidden and some not.
